docs/api/README.md

```diff
@@ -0,0 +1,159 @@
+
+#### Request
+Depending on what you want to do:
+
+- Send a `GET` request
+
+- Send a `PATCH` request with any of the following parameters (each parameter should be used either 0 or 1 times):
+    - `name` (`str`) - the activity's name/label
+    - `color` (`str`) - an RGB color with the format `<int>,<int>,<int>`, corresponding to this activity
+    - `is_everyday` (`bool`) - whether the activity should be suggested for every new day or its one-time
+
+    If you send an empty `PATCH` request, you will get an error response with the `400 Bad Request` status code
+
+#### Response
+For `GET`: response will contain a `json` of the following scheme: `{'label': <str>, 'color': [<int>, <int>, <int>],
+'is_everyday': true}`
+
+For `PATCH`: you will get a response with the `200 OK` status code
+
+If there is no activity with the `id` identifier, you will get an error response with the `404 Not Found` status code
+
+
+### `/emotions/<type_id: int>`
+Exactly like `/activities/<type_id: int>` (but for emotions)
+
+
+### `/api/days/brief`
+Get brief info about days, filled by the user you are currently logged in
+<!-- TODO: probably add some parameters limiting which days we want to retrieve (probably, dates range?) -->
+<!-- TODO: probably merge `/api/days/brief` with `/api/days`? I.e. make one endpoint with GET and POST methods -->
+
+#### Request
+Send a `GET` request
+
+#### Response
+Response will contain a `json` of the following scheme: `{'days': [{'id': <int>, 'date': <str>, 'average_color':
+[<int>, <int>, <int>]}, ...]}`, where `id` is an id of a day (you can get more info about a particular day by its id),
+`date` is the date of a day (its format is `%Y-%M-%DT00:00:00Z`) and `average_color` is an RGB color calculated as a
+proportional average between colors of emotions and activities present at a certain day.
+<!-- TODO: Remove the `T00:00:00Z` part from the date format -->
+
+
+### `/api/days`
+Add a new day
+
+#### Request
+Send a `POST` request with `Content-Type: application/x-www-form-urlencoded` with the following parameters:
+- `date` (`str`) - date of the day, formatted as `%Y-%M-%DT00:00:00Z`
+
+- `activity_type` (`int`, optional, can be used multiple times) - `type_id`s of activities of the day
+
+- `activity_proportion` (`int`, optional, must be used the same number of times `activity_type` was used) - 
+`proportion`s of activities
+
+- `emotion_type` (`int`, optional, can be used multiple times) - `type_id`s of emotions of the day
+
+- `emotion_proportion` (`int`, optional, must be used the same number of times `emotion_type` was used) - 
+`proportion`s of emotions
+
+If there is a day with the specified `date` already, you will get an error response with the `400 Bad Request` status
+code. If the numbers of entries of either `activity_type` and `activity_proportion` or `emotion_type` and
+`emotion_proportion` differ, you will get an error response with the `400 Bad Request` status code.
```

Comment:
@CoffeBee, do you find this explanation clear?

Reply:
Yes, It is quite clear